docs/plugins/testing_and_publishing/deploy.md

```diff
@@ -11,13 +11,12 @@ Once your package is listed on [PyPI] (and includes the `Framework :: napari`
 [classifier]), it will also be visible on the [napari
 hub](https://napari-hub.org/). To ensure you are providing the relevant metadata and
 description for your plugin, see the following documentation in the [napari hub
-GitHub](https://github.com/chanzuckerberg/napari-hub/tree/main/docs)’s docs
-folder:
+wiki](https://github.com/chanzuckerberg/napari-hub/wiki/Plugin-Developer's-Guide-to-the-napari-hub):
```

Comment:
I did not catch this initially, but it looks like these links are private - can the wiki be made public @DragaDoncila ?

Reply:
@melissawm yeah I get bit by this all the time. If you're signed into an account that is a member of the org, it will ask you to authorize. I think you can actually still bypass that authorize step and then you'd see the page with none of your usual permissions. But yeah, incognito is the way :laughing: